src/krb5_plugin/sssd_krb5_locator_plugin.c

```diff
@@ -569,22 +570,32 @@ krb5_error_code sssd_krb5_locator_lookup(void *private_data,
             return KRB5_PLUGIN_NO_HANDLE;
         }
 
-        PLUGIN_DEBUG("addr[%s:%s] family[%d] socktype[%d]\n",
-                     addr[c].addr, port_str, ai->ai_family, ai->ai_socktype);
+        for (ai_item = ai; ai_item != NULL; ai_item = ai_item->ai_next) {
+            ret = getnameinfo(ai_item->ai_addr, ai_item->ai_addrlen,
```

Comment:
Since the getnameinfo() result is only needed for the debug output I would suggest to only call it if ctx->debug is true.